pkg/component/kubeproxy/kube_proxy_test.go

```diff
@@ -307,13 +307,12 @@ metadata:
 				return out
 			}
 
-			configMapConntrackFixScriptName = "kube-proxy-conntrack-fix-script-40092541"
+			configMapConntrackFixScriptName = "kube-proxy-conntrack-fix-script-ebff3d39" //40092541"
```

Comment:
Seems to be a leftover from testing:
```suggestion
			configMapConntrackFixScriptName = "kube-proxy-conntrack-fix-script-ebff3d39"
```

Reply:
Good catch. My bad.